apps/backend/api/src/app/game-resources/game-resources.service.ts

```diff
@@ -74,10 +74,11 @@ export class GameResourcesService {
     return this.gameResourcesRepository
       .createQueryBuilder('gameResource')
       .innerJoinAndSelect('gameResource.game', 'game')
+      .innerJoinAndSelect('gameResource.resource', 'resource')
       .innerJoinAndSelect('game.image', 'gameImage')
       .innerJoinAndSelect('game.user', 'user')
       .innerJoinAndSelect('user.image', 'userImage')
-      .where('gameResource.resourceId = 1')
+      .where('resource.order = 1')
```

Comment:
```suggestion
      .where('resource.name = :name', { name: 'DevDollars' })
```
?
Si l'ordre change ?